### PR TITLE
fix(suite): coinjoin resume block improvements

### LINKED ISSUE #7312: Possible to resume a coinjoin session with a remembered wallet and no device
**Describe the bug**
It is now possible to click "resume" when a coinjoin session is active with a remembered wallet and there is no device.

**How to reproduce**

1. Start a Coinjoin session on a remembered wallet
1. Pause it
1. Disconnect the device
1. Try to resume the session

**Expected behavior**
It's impossible to click "resume"

### PR DESCRIPTION
**Pull request by dahaca**
## Description

* change background when resuming is not possible
* code refactors

## Related Issue

Resolve [#7312](https://github.com/trezor/trezor-suite/issues/7312)

## Screenshots:
image

### CODE REVIEW

**Review by karliatto — COMMENTED** [review]
The PR https://github.com/trezor/trezor-suite/pull/7295 was merged, @dahaca I think you still wanted to add the background changes with this PR right?

**Comment by dahaca** [comment]
@karliatto yep!

**Comment by dahaca** [comment]
@karliatto redone and ready for review!

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx [hunk 1]
```diff
@@ -259,29 +250,8 @@ export const CoinjoinStatus = ({ session, accountKey }: CoinjoinStatusProps) =>
...
-        if (isCoinjoinResumeBlocked) {
-            return (
-                <Tooltip content={<Translation id={coinjoinResumeBlockedMessageId} />}>
-                    <>
-                        <PausedInTooltipIcon icon="PAUSE" {...iconConfig} />
-                        <Translation id="TR_PAUSED" />
-                    </>
-                </Tooltip>
-            );
-        }
```
Comment (karliatto) [inline packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx:284]: I understand removing it for simplification. But where is `featureMessageContent` now? I cannot see it at all.

File: packages/suite/src/support/messages.ts [hunk 2]
```diff
@@ -3884,7 +3884,7 @@ export default defineMessages({
     },
     TR_UNAVAILABLE_COINJOIN_DEVICE_DISCONNECTED: {
         id: 'TR_UNAVAILABLE_COINJOIN_DEVICE_DISCONNECTED',
-        defaultMessage: 'Unavailable. Device is disconnected.',
+        defaultMessage: 'Unavailable. Associated device is disconnected.',
```
Comment (karliatto) [inline packages/suite/src/support/messages.ts:3887]: Opinion: I am not totally sure saying "Associated device" adds more information than only saying "Device".

File: packages/suite/src/hooks/coinjoin/useBlockedCoinjoinResume.ts [hunk 3]
```diff
@@ -13,11 +14,7 @@ export const useBlockedCoinjoinResume = () => {
     const deviceStatus = useSelector(selectDeviceState);
     const isDeviceDisconnected = deviceStatus !== 'connected';
 
-    let coinjoinResumeBlockedMessageId:
-        | 'TR_UNAVAILABLE_COINJOIN_TOR_DISABLE_TOOLTIP'
-        | 'TR_UNAVAILABLE_COINJOIN_DEVICE_DISCONNECTED'
-        | 'TR_UNAVAILABLE_COINJOIN_ACCOUNT_OUT_OF_SYNC'
-        | 'TR_UNAVAILABLE_COINJOIN_NO_INTERNET' = 'TR_UNAVAILABLE_COINJOIN_NO_INTERNET';
+    let coinjoinResumeBlockedMessageId: TranslationKey | undefined;
```
Comment (karliatto) [inline packages/suite/src/hooks/coinjoin/useBlockedCoinjoinResume.ts:28]: :cool:

File: packages/suite/src/support/messages.ts (hunk 2 above)
Comment (dahaca) [reply packages/suite/src/support/messages.ts:3887]: Perhaps, I thought it's more clear that way if you have multiple devices connected :)

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx:284]: You are right, missed it when rebased on you PR :( 

7b5bca4437af5c4d44ab299f9ccc4b1dad3bcd9e

**Review by karliatto — APPROVED** [review]
Beautiful!!! :+1:

**Review by matejkriz — APPROVED** [review]
I do like the refactor and fix, but I don't like the background color, but it's probably a matter of taste. 👌 

Please resolve the conflict and merge, so I can build https://github.com/trezor/trezor-suite/issues/7099 on top of that.

**Pull request merged**